proxy/proxy.go

```diff
@@ -235,13 +240,19 @@ func newCattleProxies(config *Config) (*proxyProtocolConverter, *cattleWSProxy)
 }
 
 type proxyProtocolConverter struct {
-	reverseProxy *httputil.ReverseProxy
-	httpsPorts   map[int]bool
+	reverseProxy        *httputil.ReverseProxy
+	httpsPorts          map[int]bool
+	apiFilterProxy      *apifilterproxy.APIFiltersHandler
+	apiFilterProxyReady bool
 }
 
 func (h *proxyProtocolConverter) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	proxyprotocol.AddHeaders(req, h.httpsPorts)
-	h.reverseProxy.ServeHTTP(rw, req)
+	if h.apiFilterProxyReady {
```

Comment:
Is this suppose to reflect the value of `manager.APIFilterProxyReady`? because the value is copied upon returning from `apifilterproxy.InitHandler()`, so changes to `manager.APIFilterProxyReady` will not be reflected here.